frontend/src/scenes/session-recordings/SessionRecordingsTable.tsx

```diff
@@ -138,6 +142,11 @@ export function SessionRecordingsTable({ personUUID, isPersonPage = false }: Ses
                         showOr
                         renderRow={(props) => <FilterRow {...props} />}
                         showNestedArrow={false}
+                        propertyFilterTaxonomicGroupTypes={
```

Comment:
No person property filters?

Reply:
Only on Postgres. Right now, the postgres query is a mix of the ORM + SQL, and getting the person filters to work here would require a big refactor (probably to only SQL). It didn't feel worth it given that postgres is on the way out.